jib-gradle-plugin/src/integration-test/resources/gradle/projects/simple/complex-build.gradle

```diff
@@ -30,7 +30,7 @@ jib {
     }
   }
   container {
-    useCurrentTimestamp = true
+    creationTime = 'USE_CURRENT_TIMESTAMP'
```

Comment:
Oops, I think some of my comments got lost. Is this being tested explicitly (using the keyword in the new config)? For Gradle, I couldn't find in this PR.

Reply:
Yes https://github.com/GoogleContainerTools/jib/blob/5d3d0d673b25c5533a39f1780267d8bf4730df5d/jib-gradle-plugin/src/integration-test/resources/gradle/projects/simple/build.gradle#L23

Reply:
I think that's handled here https://github.com/GoogleContainerTools/jib/pull/1888/files#diff-4e0bf50db814ac7da352939e82b1240eR310